File: t001296.cpp
#include <stdio.h>

main(){
    int a1, a2, a3;
    while(1){
        scanf("%d", &a1);
        if(a1 == 0){
            break;
        }
        scanf("%d", &a2);
        scanf("%d", &a3);
        if(a1 == 0 || a2 == 0 || a3 == 0){
            printf("oh,my god!\n");
        }
        else{
            if(a1 + a2 > a3 &&
               a1 + a3 > a2 &&
               a2 + a3 > a1){
                   printf("Great,you are genius!\n");
               }
            else{
                printf("oh,my god!\n");
            }
        }
    }
}

File: t001296.py
while 1:
    n = [int(i) for i in input().split()]
    if len(n) == 1 and n[0] == 0:
        break
    if n[0] == 0 or n[1] == 0 or n[2] == 0:
        print("oh,my god!", end="\n")
    else:
        if n[0] + n[1] > n[2] and\
           n[0] + n[2] > n[1] and\
           n[1] + n[2] > n[0]:
            print("Great,you are genius!", end="\n")
        else:
            print("oh,my god!", end="\n")
